spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -28,6 +28,7 @@
 		<prometheus-rsocket-spring.version>1.3.0</prometheus-rsocket-spring.version>
 		<java-cfenv.version>2.3.0</java-cfenv.version>
 		<spring-cloud-services-starter-config-client.version>3.2.1.RELEASE</spring-cloud-services-starter-config-client.version>
+		<oracle-jdbc.version>19.8.0.0</oracle-jdbc.version>
```

Comment:
@dturanski this reminds me that the task samples are [still pointing to v20](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/main/timestamp-batch/pom.xml#L20).

Reply:
v21 actually, I didn't change the version in the samples just to try it. No issues AFAICT.

Reply:
Ack. Is the intention to leave them v21? Wdyt about leaving one at v21 and one at v19? The reason is that someone may see the difference in version number while debugging an issue later and think that could be it and go down that road. Maybe add comment on the v21 one just saying its intentonal. 

At the end of the day... its not a big deal. Just putting my thoughts here.